src/webgpu/shader/execution/memory_model/memory_model_setup.ts

```diff
@@ -959,12 +983,15 @@ export function buildTestShader(
   testType: TestType
 ): string {
   let memoryTypeCode;
+  let isStorageClass = false;
```

Comment:
FYI.  We renamed "storage class" to "address space", so you can rename this  isStorageAS or something like that.